ferretdb/ferretdb.go

```diff
@@ -55,6 +49,27 @@ type Config struct {
 	TigrisURL          string
 }
 
+// ListenerConfig represents listener configuration.
```

Comment:
FWIW, #38 explicitly states that embeddable `ferretdb` package is not in the scope. But I guess we could keep it if it works